docs/context.rst

```diff
@@ -56,10 +56,20 @@ Lazy evaluation
     evaluation is lazy. It stops immediately when we know the
     final result.
 
-Expression skipping
+Boolean operations with CannotDecide
     When a dimension or outcome of the operation is not defined,
-    the expression is skipped over.
+    the expression is treated as CannotDecide.
 
+    Boolean operations with CannotDecide
+
+============  ===  ============  ============
+CannotDecide  and  True          CannotDecide
+CannotDecide  and  False         False
+CannotDecide  or   True          True
+CannotDecide  or   False         CannotDecide
+CannotDecide  and  CannotDecide  CannotDecide
+CannotDecide  or   CannotDecide  CannotDecide
+============  ===  ============  ============
```

Comment:
Looking at the table I'm missing the `==` before the last column. It's a bit confusing. What about just writing theses examples as expressions?

    CannotDecide and True == CannotDecide